src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultSorting = (a, b) => {
+    const stringA = String(a);
+    const stringB = String(b);
+
+    if (stringA > stringB) {
+      return 1;
+    } else if (stringA === stringB) {
+      return 0;
+    } else {
+      return -1;
+    }
+  };
```

Comment:
try not to use `else if` nesting

You can use early return. Example

```
if (a === b) {
  return 'equal'
}

if (a === c) {
  return 'sth'
}

return 'sb'
```

Reply:
got it, thanks